configure

```diff
@@ -15989,6 +15989,13 @@ if test x"$with_bfd" != "xno"; then :
 fi
       if test -z "$BFD_LIB_DIR"; then :
   BFD_LIB_DIR="/opt/local/lib"
+fi ;; #(
+  *-*openbsd*) :
+    if test -z "$BFD_INCLUDE_DIR"; then :
+  BFD_INCLUDE_DIR="/usr/local/include"
+fi
+      if test -z "$BFD_LIB_DIR"; then :
+  BFD_LIB_DIR="/usr/local/lib"
```

Comment:
I believe this is not such a good idea. I don't think this needs a fixing _at all_.
OpenBSD has all ports installed in ``/usr/local``. So ``OpenBSD`` should provide this information. Either by setting ``BFD_LIB_DIR`` / ``BFD_INCUDE_DIR`` or ``LIBRARY_PATH`` / ``CPATH`` for us.
I would strongly argue against configuring something like this conditional on the OS.
If you really think the configure script should discover the bfd library without outside help you may search for it in common places (``/usr/``, ``/usr/local``, ``/opt``). But as I said I don't think this needs fixing in the OCaml build system. I am regularly maintaining the OCaml port on OpenBSD. It already has 
``` make
CFLAGS +=	-I${LOCALBASE}/include
LDFLAGS +=	-L${LOCALBASE}/lib
```
in it. Everyone else should set ``LIBRARY_PATH`` and ``CPATH`` when building projects supposed to find stuff in ``/usr/local``.

Reply:
Note that somehow what @madroach mentions is the same conclusion we eventually arrived to thanks to @ygrek in `opam`, see [here for a discussion](https://github.com/ocaml/opam-repository/issues/3093#issuecomment-312929998).